src/gen-ts.ts

```diff
@@ -136,6 +139,21 @@ function analyzerToAst(
   return tsDocs;
 }
 
+/**
+ * Analyzer always returns fully specified URLs with a protocol and an absolute
+ * path (e.g. "file:/foo/bar"). Return just the file path, relative to our
+ * project root.
+ */
+function analyzerUrlToRelativePath(
+    analyzerUrl: string, rootDir: string): string {
+  const parsed = url.parse(analyzerUrl);
+  if (parsed.protocol !== 'file:' || parsed.path === undefined) {
+    console.error(`Expected valid file URL, got: ${analyzerUrl}`);
```

Comment:
This is legit, I don't think we should warn here

Reply:
Done. Moved warning to call sites and made more informative.